post-archive/post_archive.rb

```diff
@@ -238,6 +238,11 @@ def checkForTranscode(path, filename)
   pathToFile = File.join(path, filename)
   outputPathToFile = File.join(TMP_PATH, pathToFile)
 
+  if (File.zero?(pathToFile))
+    BigBlueButton.logger.info( "Video file #{pathToFile} has size zero, skipping...")
```

Comment:
Not really that important but for consistency…
```suggestion
    BigBlueButton.logger.info("Video file #{pathToFile} has size zero, skipping...")
```

Reply:
Alas, whether there is a space or not before the logging string is not consistent within the script (and yes, that is my fault).

Reply:
Actually… the other log statements have the same inconsistency. So, in that, it is consistent :see_no_evil: